### PR TITLE
Dashboard

### PR DESCRIPTION
**Pull request by BipinKalra**
- [ ] Dashboard layout done

- [ ] nav hover fixed

### CODE REVIEW

File: examples/views/hb/dashboard.hbs [hunk 1]
```diff
@@ -0,0 +1,66 @@
+<div class="a-hb">
+  <div>
+    {{>nav-hb}}
+    <div class="row justify-content-center mx-0">
+      {{>sidenav}}
+      <div class="offset-0 offset-md-1 col-11 px-lg-5 px-md-4 px-0 dashboard pt-5">
+        <div class="position-relative mb-5 banner">
+          <div class="banner-nav h-100 w-10 position-absolute tl flex-col all-center pointer">
+            <span style="font-size: 50px;"><a href="#" class="white extra-bold">&lang;</a></span>
```
Comment (vdvibhu20) [inline examples/views/hb/dashboard.hbs]: remove inline styling

File: examples/components/hb/dashboard-metrics-card.hbs [hunk 2]
```diff
@@ -0,0 +1,42 @@
+{{!-- <div class="border-card py-5 px-4">
```
Comment (vdvibhu20) [inline examples/components/hb/dashboard-metrics-card.hbs]: remove comments

File: examples/components/hb/dashboard-reg-contests-card.hbs [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+  </div>
+
+  <div class="divider-h px-sm-5 px-4"></div>
+
+  <div class="row no-gutters justify-content-between align-items-center hover-grey py-4 px-sm-5 px-4">
+    <div class="col-lg-8 col-md-6 col-7">
+      <div class="row mx-0 align-items-center">
+        <div class="px-0 col-4 col-xl-2">
+          <div class="border-card pt-2 pb-3 px-4 flex-col all-center orange border-orange">
+            <div class="extra-bold" style="font-size: 36px;">26</div>
```
Comment (vdvibhu20) [inline examples/components/hb/dashboard-reg-contests-card.hbs]: inline style

File: examples/views/hb/dashboard.hbs [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+            {{>dashboard-reg-contests-card}}
+          </div>
+          <div class="col-md-4 col-12">
+            {{!-- Performance Graph Card to be placed here --}}
+            <div class="border-card p-5 mb-5 mt-md-0 mt-5">
+              <div class="row mx-0 justify-content-between">
+                <h4>Performance Graph</h4>
+                <div class="card-md">Past 6 Months</div>
+              </div>
+              <div class="bg-grey mt-4" style="height: 300px;"></div>
```
Comment (vdvibhu20) [inline examples/views/hb/dashboard.hbs]: inline style

File: examples/components/hb/dashboard-reg-contests-card.hbs [hunk 5]
```diff
@@ -0,0 +1,133 @@
+<div class="border-card py-5 px-0 mt-5">
+  <div class="row justify-content-between align-items-center mx-0 px-sm-5 px-4">
+    <h3 class="mb-4 mt-4">Registered Contests</h3>
+    <button class="button-solid button-orange font-sm my-3">View All</button>
+  </div>
+  <div class="row no-gutters justify-content-between align-items-center hover-grey py-4 px-sm-5 px-4 mt-4">
```
Comment (vdvibhu20) [inline examples/components/hb/dashboard-reg-contests-card.hbs]: `<div class="row no-gutters  hover-grey py-4 px-sm-5 px-4">
                <div class="li-left">
                  <div class="">
                      <div class="border-card pt-2 pb-3 px-4 t-align-c orange border-orange">
                        <div class="extra-bold" style="font-size: 36px;">26</div>
                        <div class="font-sm bold mt-neg-10">JUNE</div>
                      </div>
                    </div><div class="">
                      <h5 class="bold mb-1">APAT June 2019</h5>
                      <div>Admission Contest</div>
                    </div><div class="row mx-0 align-items-center">
                    
                    
                  </div>
                </div>
                <div class="t-align-r li-right">
                  <div>
                    <div><a href="#">Remind Me</a></div>
                    <h6 class="mt-2"><a href="#" class="orange">View Contest<i class="fas fa-arrow-right ml-2"></i></a>
                    </h6>
                  </div>
                </div>
              </div>`

File: examples/components/hb/dashboard-reg-contests-card.hbs [hunk 6]
```diff
@@ -0,0 +1,133 @@
+<div class="border-card py-5 px-0 mt-5">
+  <div class="row justify-content-between align-items-center mx-0 px-sm-5 px-4">
```
Comment (vdvibhu20) [inline examples/components/hb/dashboard-reg-contests-card.hbs]: here also we can  use li-left li-right

File: examples/components/hb/dashboard-practice-card.hbs [hunk 7]
```diff
@@ -0,0 +1,56 @@
+<div class="border-card p-0">
```
Comment (vdvibhu20) [inline examples/components/hb/dashboard-practice-card.hbs]: Screenshot 2019-07-26 at 10 31 11 PM
extra top-bottom spaceing

File: examples/components/hb/dashboard-reg-contests-card.hbs (hunk 5 above)
Comment (BipinKalra) [reply examples/components/hb/dashboard-reg-contests-card.hbs]: li-left and li-right mein kuchh screen sizes mein fat rha tha.

File: examples/components/hb/dashboard-dcb-card.hbs [hunk 8]
```diff
@@ -0,0 +1,50 @@
+<div class="border-card p-0 h-100">
```
Comment (vdvibhu20) [inline examples/components/hb/dashboard-dcb-card.hbs]: Screenshot 2019-07-27 at 2 15 46 PM
padding is too much

File: examples/components/hb/dashboard-leaderboard.hbs [hunk 9]
```diff
@@ -0,0 +1,198 @@
...
+            <div class="img-desc align-items-center">
+              <img class="round s-30x30" src="/images/placeholder.gif" alt="">
+              <div class="description ml-3">
+                Siddharth Jain
+              </div>
+            </div>
+          </td>
+          <td class="pb-4 pt-4">2345</td>
+        </tr>
+        <tr class="hover-grey">
```
Comment (vdvibhu20) [inline examples/components/hb/dashboard-leaderboard.hbs]: Screenshot 2019-07-29 at 12 40 08 PM
padding top and bottom in hover

File: examples/components/hb/dashboard-interview-card.hbs [hunk 10]
```diff
@@ -0,0 +1,19 @@
+<div class="border-card p-5 bg-gradient-navy white h-100">
```
Comment (vdvibhu20) [inline examples/components/hb/dashboard-interview-card.hbs]: Screenshot 2019-07-29 at 12 41 25 PM
card's height is greater than in designs.

File: examples/views/hb/dashboard.hbs [hunk 11]
```diff
@@ -0,0 +1,66 @@
...
+          </div>
+          <div class="col-lg-4 col-md-5 col-sm-8 col-10">
+            {{>dashboard-practice-card}}
+          </div>
+          <div class="col-lg-4 col-md-5 col-sm-8 col-10">
+            {{>dashboard-practice-card}}
+          </div>
+        </div>
+        <div class="row mx-0 my-5">
+          <div class="col-md-8 col-12 pl-md-0">
```
Comment (vdvibhu20) [inline examples/views/hb/dashboard.hbs]: `.pl-md-0`?

File: examples/views/hb/dashboard.hbs [hunk 12]
```diff
@@ -0,0 +1,66 @@
...
+            {{>dashboard-badge-card}}
+          </div>
+          <div class="col-md-4 col-12 pr-md-0 mt-md-0 mt-5">
```
Comment (vdvibhu20) [inline examples/views/hb/dashboard.hbs]: `pr-md-0`?

File: examples/views/hb/dashboard.hbs [hunk 13]
```diff
@@ -0,0 +1,66 @@
+<div class="a-hb">
+  <div>
+    {{>nav-hb}}
+    <div class="row justify-content-center mx-0">
+      {{>sidenav}}
+      <div class="offset-0 offset-md-1 col-11 px-lg-5 px-md-4 px-0 dashboard pt-5">
+        <div class="position-relative mb-5 banner">
+          <div class="banner-nav h-100 w-10 position-absolute tl flex-col all-center pointer">
```
Comment (vdvibhu20) [inline examples/views/hb/dashboard.hbs]: this part in css`h-100 w-10 position-absolute tl flex-col all-center pointer`

**Pull request merged**